pkg/target/types.go

```diff
@@ -53,7 +54,8 @@ type Container struct {
 	ReqCPU    float64
 	ReqMemory float64
 
-	QPS Resource
+	QPS          Resource
+	ResponseTime *Resource
```

Comment:
Note that: the `ContainerTemplate` is shared by all the `Container` s in all the `Pods`.  It is safer to use struct, instead of pointer.

The `virtualCluster` can accept actions to adjust the resources. Even now the `QPS/ResponseTime` cannot be adjusted, it is possible that they can be adjusted.  If they can be adjusted, then the change on one container will affect all the other containers with the same `ContainerTemplate`.